blocks/collections-content-source-block/sources/content-api-collections.js

```diff
@@ -1,65 +1,72 @@
 import axios from "axios";
-import { CONTENT_BASE, ARC_ACCESS_TOKEN } from "fusion:environment";
-import getResizedImageData from "@wpmedia/resizer-image-block";
+import { CONTENT_BASE, ARC_ACCESS_TOKEN, RESIZER_APP_VERSION } from "fusion:environment";
+
+import { signImagesInANSObject } from "@wpmedia/arc-themes-components";
+import { fetch as resizerFetch } from "@wpmedia/signing-service-content-source-block";
 
 const params = {
 	_id: "text",
+	"arc-site": "text",
```

Comment:
`arc-site` shouldn't be set as a param - this parameter is automatically added to the query object by Fusion - we do not want to expose this to the user for entry